lib/utils.py

```diff
@@ -20,7 +21,10 @@
 RESERVATION_NOT_FOUND_CODE = 400620389
 
 
-def make_request(method: str, site: str, headers: JSON, info: JSON, max_attempts=20) -> JSON:
+def set_sleep_duration(min_duration: float, max_duration: float) -> float:
```

Comment:
Could the name of this function be changed to `random_sleep_duration`? That way it makes more sense as it isn’t setting anything within it.